ga4gh/datamodel/reads.py

```diff
@@ -350,7 +352,8 @@ def convertReadAlignment(self, read):
             read.reference_id)
         ret.alignment.position.position = read.reference_start
         ret.alignment.position.strand = \
```

Comment:
The ternary operator is a bit awkward here with the style requirements for line breaking. The following suggestion is easier style-wise, and is quite straightforward:
```python
ret.alignment.position.strand =  protocol.Strand.POS_STRAND
if SamFlags.isFlagSet(read.flag, SamFlags.REVERSED):
     ret.alignment.position.strand =  protocol.Strand.NEG_STRAND
```